zookeeper-server/src/main/java/org/apache/zookeeper/backup/BackupException.java

```diff
@@ -0,0 +1,7 @@
+package org.apache.zookeeper.backup;
```

Comment:
Might be a better idea to move this into a separate package such as `...backup.exception` or something along those lines.